dali/dfuplus/dfuplus.cpp

```diff
@@ -739,25 +756,34 @@ int CDfuPlusHelper::despray()
     }
     else
     {
-        if(dstip != nullptr || dstfile != nullptr)
-            throw MakeStringException(-1, "dstip/dstfile and dstxml can't be used at the same time");
+        if(!isEmptyString(dstip) || !isEmptyString(dstfile) || !isEmptyString(dstplane))
+            throw MakeStringException(-1, "dstip/dstfile/dstplane and dstxml can't be used at the same time");
         StringBuffer buf;
         buf.loadFile(dstxml);
         int len = buf.length();
         xmlbuf.setBuffer(len, buf.detach(), true);
     }
 
     if(dstxml == nullptr)
-        info("\nDespraying %s to host %s file %s\n", srcname, dstip, dstfile ? dstfile : "");
+    {
+        if(dstplane)
+            info("\nDespraying %s to storage plane %s file %s\n", srcname, dstplane, dstfile ? dstfile : "");
+        else
+            info("\nDespraying %s to host %s file %s\n", srcname, dstip, dstfile ? dstfile : "");
+    }
     else
         info("\nDespraying %s\n", srcname);
 
     Owned<IClientDespray> req = sprayclient->createDesprayRequest();
     req->setSourceLogicalName(srcname);
     if(dstxml == nullptr)
     {
-        req->setDestIP(dstip);
+        if (!isEmptyString(dstplane))
```

Comment:
I'd prefer to leave as is because (1) the extra calls isn't going to impact performance of the dfuplus cli (2) if I change for dstplane, I should be doing the same for the other similar ones and I'd prefer not to make to code changes.

Reply:
happy for these changes to be made later - otherwise it is hard to know where to stop